pubspec.yaml

```diff
@@ -85,17 +87,17 @@ flutter:
   # in this "flutter" section. Each entry in this list should have a
   # "family" key with the font family name, and a "fonts" key with a
   # list giving the asset and other descriptors for the font. For
-  fonts:
```

Comment:
Did you also removed these files from assets folder?

Reply:
yes, i did